tls_icon.php

```diff
@@ -5,6 +5,9 @@ class tls_icon extends rcube_plugin
 	private $message_headers_done = false;
 	private $icon_img;
 	private $rcmail;
+	private $postfix_tls_regex = "/\(using (TLS.*)\) \(/im";
```

Comment:
They could be const

Reply:
You read my mind. I definetly wanted to do this, but then got distracted with the tests and forgot. Thanks!